### DISCUSSION TITLE
Sum specific table column

### DISCUSSION THREAD

**Discussion by eudovic** [Laravel-Backpack/community-forum#715] [Q&A (Help)] [answered]
Is there a way to obtain the sum of the column values ​​in the list operation within the header and footer of a table?
<img width="789" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/31167386/02fe19d7-22e9-4643-ae06-7e2763cdbfe2">

**Comment by pxpm** [comment, marked as answer]
Hey @eudovic thanks for the question. 

We don't have a built in way to do it. We discussed it a little bit here: https://github.com/Laravel-Backpack/CRUD/issues/1441

One of the community members shared a little snippet he uses to add the sum of a column: https://gist.github.com/breizhwave/545a361ed941d7b449e7a9a07bf97055

To avoid overwriting the whole `list.blade.php` you can add in as a general script https://backpackforlaravel.com/docs/6.x/base-how-to#add-custom-javascript-to-all-admin-panel-pages, but only register it when an element with attribute `bp-section="crud-operation-list` exist on page. 
Alternatively you can add it as a widget directly on the operation: https://backpackforlaravel.com/docs/6.x/crud-how-to#how-to-add-css-or-js-to-a-page-or-operation

Hope that helps.

Cheers
